### PR TITLE
Added Supplementum Aristotelicum I

### PR DESCRIPTION
**Pull request by planatheisa**
First upload. Checking.

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `coverage`         |    -0.01 |
| `metadata_passing` |     2    |
| `metadata_total`   |     2    |
| `nodes_count`      |     3    |
| `texts_total`      |     1    |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +38615 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0644/__cts__.xml`                          | New      |
| `./data/tlg0644/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg0644/__cts__.xml`                          | Passing  |
| `./data/tlg0644/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | Failing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/1721)*

**Comment by sonofmun** [comment]
```
>>>> Testing /tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml
>>>>> Epidoc DTD validation failed
>>>>>> error: value of attribute "cause" is invalid [In (L112 C52); (L202 C49); (L422 C49); (L473 C49); (L758 C54); (L876 C50); (L979 C49); (L1042 C81); (L1045 C83); (L1048 C88); (L1049 C74); (L1207 C74); (L1321 C83); (L1561 C83); (L1607 C90); (L1658 C63); (L2013 C52); (L2148 C78); (L2158 C34); (L2268 C65); (L2311 C77); (L2470 C87); (L2587 C49); (L2626 C34); (L2714 C64); (L2762 C49); (L2772 C71); (L2781 C78); (L2821 C81); (L3276 C34); (L3348 C54); (L3380 C34); (L3430 C58); (L3518 C89); (L3616 C34); (L3632 C34); (L3776 C34); (L4049 C82); (L4076 C81); (L4077 C56); (L4079 C49); (L4080 C63); (L4081 C71); (L4082 C68); (L4083 C65); (L4084 C49); (L4085 C49); (L4086 C91); (L4107 C49); (L4194 C49); (L4261 C84); (L4469 C49); (L4469 C92); (L4814 C49); (L4858 C34); (L5000 C54); (L5154 C34); (L5368 C49); (L5489 C54); (L5860 C67); (L5862 C50); (L5875 C54); (L6084 C62); (L6114 C49); (L6122 C92); (L6377 C49); (L6527 C87); (L6529 C49); (L6530 C77); (L6737 C87); (L6739 C49); (L6840 C71); (L6842 C57); (L6843 C38); (L6847 C65); (L6890 C61); (L6893 C42); (L6894 C62); (L6895 C77); (L6897 C49); (L6898 C69); (L6938 C57); (L6939 C91); (L6943 C73); (L6944 C91); (L6946 C76); (L6993 C84); (L6995 C49); (L6996 C49); (L6997 C57); (L6998 C77); (L6999 C89); (L7001 C49); (L7046 C49); (L7046 C84); (L7048 C34); (L7051 C57); (L7052 C30); (L7053 C50); (L7053 C83); (L7054 C95); (L7055 C76); (L7056 C93); (L7058 C86); (L7060 C50); (L7061 C71); (L7109 C55); (L7110 C34); (L7110 C66); (L7112 C64); (L7113 C42); (L7113 C76); (L7114 C72); (L7115 C50); (L7116 C76); (L7118 C34); (L7119 C56); (L7120 C74); (L7128 C34); (L7167 C84); (L7168 C50); (L7171 C34); (L7171 C75); (L7172 C73); (L7173 C50); (L7174 C57); (L7174 C96); (L7176 C50); (L7177 C69); (L7178 C80); (L7227 C55); (L7228 C79); (L7229 C50); (L7231 C34); (L7231 C71); (L7232 C42); (L7233 C50); (L7233 C82); (L7234 C55); (L7237 C51); (L7238 C42); (L7240 C50); (L7240 C92); (L7242 C76); (L7243 C89); (L7245 C56); (L7246 C68); (L7279 C85); (L7289 C87); (L7292 C72); (L7294 C62); (L7295 C83); (L7297 C74); (L7299 C50); (L7299 C84); (L7301 C34); (L7302 C50); (L7303 C68); (L7346 C71); (L7348 C86); (L7350 C42); (L7351 C63); (L7352 C94); (L7354 C57); (L7355 C83); (L7394 C55); (L7399 C67); (L7402 C82); (L7404 C42); (L7405 C63); (L7406 C81); (L7408 C42); (L7410 C51); (L7411 C75); (L7412 C50); (L7413 C73); (L7445 C92); (L7458 C55); (L7461 C91); (L7463 C79); (L7465 C42); (L7466 C63); (L7467 C94); (L7469 C83); (L7471 C34); (L7472 C49); (L7472 C81); (L7519 C85); (L7521 C75); (L7523 C42); (L7525 C87); (L7527 C66); (L7556 C64); (L7576 C86); (L7578 C50); (L7580 C49); (L7615 C51); (L7616 C69); (L7619 C49); (L7620 C85); (L7622 C78); (L7627 C70); (L7679 C71); (L7689 C67); (L7692 C68); (L7695 C58); (L7696 C86); (L7737 C89); (L7739 C49); (L7740 C76); (L7805 C60); (L7806 C34); (L7808 C55); (L7808 C86); (L7810 C49); (L7811 C66); (L7812 C79); (L7852 C61); (L7857 C62); (L7861 C66); (L7865 C55); (L7866 C73); (L7909 C79); (L7911 C88); (L7918 C49); (L7918 C78); (L7921 C54); (L7922 C82); (L7924 C42); (L7925 C79); (L7927 C34); (L7973 C49); (L7977 C49); (L7978 C82); (L7980 C49); (L7981 C87); (L7983 C49); (L7984 C73); (L7985 C76); (L7986 C55); (L7987 C79); (L8035 C49); (L8037 C49); (L8038 C65); (L8039 C65); (L8040 C67); (L8042 C49); (L8043 C72); (L8045 C34); (L8046 C57); (L8084 C49); (L8087 C49); (L8090 C67); (L8091 C49); (L8093 C57); (L8095 C57); (L8096 C83); (L8098 C57); (L8098 C88); (L8100 C85); (L8102 C57); (L8103 C69); (L8106 C57); (L8108 C57); (L8109 C85); (L8161 C85); (L8165 C49); (L8166 C42); (L8167 C88); (L8168 C62); (L8169 C34); (L8170 C55); (L8172 C50); (L8173 C34); (L8173 C66); (L8174 C86); (L8176 C49); (L8177 C49); (L8202 C55); (L8203 C79); (L8209 C49); (L8210 C49); (L8225 C60); (L8226 C34); (L8227 C50)]
```

File: data/tlg0644/__cts__.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0644">
+  <ti:groupname xml:lang="eng">Aristophaes of Byzantium</ti:groupname>
```
Comment (sonofmun) [inline data/tlg0644/__cts__.xml]: Name is misspelled.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |       +3 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +38615 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0644/__cts__.xml`                          | New      |
| `./data/tlg0644/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg0644/__cts__.xml`                          | Passing  |
| `./data/tlg0644/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/1888)*

File: data/tlg0644/__cts__.xml (hunk 1 above)
Comment (planatheisa) [inline data/tlg0644/__cts__.xml]: Any more or can I commit again?

**Comment by sonofmun** [comment]
In tlg0644.tlg001.1st1K-grc1.xml: 
- [ ] `<publicationStmt>/<date>` should be 2017 instead of 2016.
- [ ] I think we need to break the text down into smaller parts. Right now it is 38K words and only two chapters. There are sections with numbers in parentheses, e.g., (1). I would run a regular expression over this and change those all to `<div type="textpart" subtype="section" n="#">`. Let me know if you need help with this.

**Comment by planatheisa** [comment]
OK

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |       +3 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +38615 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0644/__cts__.xml`                          | New      |
| `./data/tlg0644/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg0644/__cts__.xml`                          | Passing  |
| `./data/tlg0644/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/1890)*

**Comment by planatheisa** [comment]
Should I get rid of the <div subtype="book"> here also? And add a <div subtype="chapter" n="prae"> for whatever it is that comes before chapter 1? Unfortunately I can't access the scan at Hathi Trust, so I can't see what it actually looks like. On that note: Should I add <ref> for a scan even if you can't freely access it, as is the case here?

**Comment by planatheisa** [comment]
I'll push the change for now, so I can safely work on the other file. If any more changes are needed, I'll do another commit.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `coverage`         |    -0.01 |
| `metadata_passing` |     2    |
| `metadata_total`   |     2    |
| `nodes_count`      |   780    |
| `texts_total`      |     1    |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +38615 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0644/__cts__.xml`                          | New      |
| `./data/tlg0644/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg0644/__cts__.xml`                          | Passing  |
| `./data/tlg0644/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | Failing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/1891)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |     +780 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +38615 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0644/__cts__.xml`                          | New      |
| `./data/tlg0644/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg0644/__cts__.xml`                          | Passing  |
| `./data/tlg0644/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/1896)*

**Comment by sonofmun** [comment]
We prefer an archive.org scan for precisely this reason, but if the only scan available is a Hathi Trust scan, we should use that.

**Comment by sonofmun** [comment]
And, yes, I would get rid of the subtype="book" and add the subtype="chapter" n="praef" to the file.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |     +780 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +38615 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0644/__cts__.xml`                          | New      |
| `./data/tlg0644/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg0644/__cts__.xml`                          | Passing  |
| `./data/tlg0644/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0644/tlg001/tlg0644.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/1911)*

**Comment by planatheisa** [comment]
@sonofmun 

Can you check again?

**Pull request merged**